ursa/local_manager.py

```diff
@@ -8,12 +8,16 @@ def __init__(self):
         self.graph_dict = {}
         self._graph_config = {}
         self._transaction_id = 0
+        self.versions_to_store = 5
 
     def update_transaction_id(self):
         """Updates the transaction ID with that of the global graph manager."""
         pass
 
-    def create_graph(self, graph_id, directed=False, new_transaction=True):
+    def create_graph(self,
```

Comment:
Why did this change? This looks like maybe an auto-formatting thing.

Reply:
Resolved all these; sorry about the formatting - it was from when I added arguments and forgot to revert the formatting when deleting those arguments.